spec/spec_helper.rb

```diff
@@ -45,3 +44,8 @@
       to_return(status: 200, body: "[{\"number\": \"7.0.0\"}, {\"number\": \"6.1.6\"}]", headers: {})
   end
 end
+
+def with_env(env_hash, &block)
+  stub_const("ENV", env_hash)
```

Comment:
Here probably we can do something more sophisticated but for now, probably is fine as it is, don't you think?

```suggestion
  stub_const("ENV", {}.merge(env_hash))
```

Reply:
In Ruby 2.X, `ENV` doesn't have the `merge` method making it `hash` and then using it works